src/compose/utils.ts

```diff
@@ -373,6 +373,15 @@ export function addFeaturesFromLabels(
 		'io.balena.features.sysfs': () => service.config.volumes.push('/sys:/sys'),
 		'io.balena.features.procfs': () =>
 			service.config.volumes.push('/proc:/proc'),
+		'io.balena.features.gpu': () =>
```

Comment:
should this be device-type dependent @robertgzr ?

Reply:
now that you mention it. yeah probably should be...

Reply:
it's also specific to docker api v1.40, but not sure if we care about that?